cpu/stm32/Kconfig

```diff
@@ -15,6 +15,7 @@ config CPU_STM32
     select HAS_PERIPH_RTT_OVERFLOW
     select HAS_PERIPH_UART_MODECFG
     select HAS_PERIPH_UART_NONBLOCKING
+    select HAS_PERIPH_UART_RS485
```

Comment:
Should only be selected by the specific families (like done in `Makefile.features`), please check the specific Kconfig files in `cpu/stm32/kconfigs/<family>/Kconfig`.

Reply:
@leandrolanzieri done.